kalite/topic_tools/__init__.py

```diff
@@ -58,10 +60,10 @@ def get_topic_tree(force=False, annotate=False, channel=settings.CHANNEL):
 
     if annotate:
         if settings.DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP and not force:
-            topics = softload_json(TOPICS_FILEPATHS.get(channel) + ".cache", logger=logging.debug, raises=False)
+            topics = softload_json(TOPICS_FILEPATHS.get(channel) + "_" + language + ".cache", logger=logging.debug, raises=False)
```

Comment:
So if we don't set `annotate` to `True` we won't generate the topic tree for the language?